fabfile.py

```diff
@@ -272,6 +272,46 @@ def upgrade_binary(conn):
     conn.run(f'dtach -n `mktemp -u /tmp/dtach.XXXX` sh /home/ubuntu/cmd.sh')
 
 
+# ======================================================================================
+#                                      validator rotation
+# ======================================================================================
+
+@task
+def send_cli_binary(conn):
```

Comment:
u can use `send_zip` here

Reply:
Done, also made the `send_binary` function do the same, and fixed the order of arguments in `send_zip` (the only usages had the zip file and binary in the wrong order?).

Reply:
lul